py3status/modules/imap.py

```diff
@@ -9,7 +9,9 @@
     hide_if_zero: don't show on bar if 0
     imap_server: IMAP server to connect to
     mailbox: name of the mailbox to check
+    maildir: full path of your maildir location
     new_mail_color: what color to output on new mail
+    offline: set to 1 will only check for local maildir
```

Comment:
Maybe instead of this option just see if we have imap_server set, if imap_server == '<IMAP_SERVER>' then nothing remote to check.

Less config options is always nice.